### PR TITLE
Update gcp broker example docs

### PR DESCRIPTION
**Pull request by liu-cong**
Fixes #

## Proposed Changes

- Update broker URL to match the recent brokercell change.
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: docs/examples/gcpbroker/README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ To show the various types of events you can send, you will make three requests:
    `type:greeting`, run the following in the SSH terminal:
 
    ```sh
-   curl -v "http://broker-ingress.cloud-run-events.svc.cluster.local/cloud-run-events-example/test-broker" \
+   curl -v "http://default-brokercell-ingress.cloud-run-events.svc.cluster.local/cloud-run-events-example/test-broker" \
      -X POST \
      -H "Ce-Id: say-hello" \
      -H "Ce-Specversion: 0.3" \
```
Comment (capri-xiyue) [inline docs/examples/gcpbroker/README.md:82]: Can you change this to `1.0`

File: docs/examples/gcpbroker/README.md [hunk 2]
```diff
@@ -102,7 +102,7 @@ To show the various types of events you can send, you will make three requests:
    `source:sendoff`, run the following in the SSH terminal:
 
    ```sh
-   curl -v "http://broker-ingress.cloud-run-events.svc.cluster.local/cloud-run-events-example/test-broker" \
+   curl -v "http://default-brokercell-ingress.cloud-run-events.svc.cluster.local/cloud-run-events-example/test-broker" \
      -X POST \
      -H "Ce-Id: say-goodbye" \
      -H "Ce-Specversion: 0.3" \
```
Comment (capri-xiyue) [inline docs/examples/gcpbroker/README.md:108]: similar `1.0`

File: docs/examples/gcpbroker/README.md [hunk 3]
```diff
@@ -129,7 +129,7 @@ To show the various types of events you can send, you will make three requests:
    terminal:
 
    ```sh
-   curl -v "http://broker-ingress.cloud-run-events.svc.cluster.local/cloud-run-events-example/test-broker" \
+   curl -v "http://default-brokercell-ingress.cloud-run-events.svc.cluster.local/cloud-run-events-example/test-broker" \
      -X POST \
      -H "Ce-Id: say-hello-goodbye" \
      -H "Ce-Specversion: 0.3" \
```
Comment (capri-xiyue) [inline docs/examples/gcpbroker/README.md:135]: similar `1.0`

File: docs/examples/gcpbroker/README.md [hunk 4]
```diff
@@ -274,7 +274,7 @@ consumers are back.
 1. Send an event that has the `type:greeting` and the`source:sendoff`:
 
    ```sh
-   curl -v "http://broker-ingress.cloud-run-events.svc.cluster.local/cloud-run-events-example/test-broker" \
+   curl -v "http://default-brokercell-ingress.cloud-run-events.svc.cluster.local/cloud-run-events-example/test-broker" \
      -X POST \
      -H "Ce-Id: say-hello-goodbye" \
      -H "Ce-Specversion: 0.3" \
```
Comment (capri-xiyue) [inline docs/examples/gcpbroker/README.md:280]: similar `1.0`

File: docs/examples/gcpbroker/README.md (hunk 1 above)
Comment (liu-cong) [reply docs/examples/gcpbroker/README.md:82]: The event-display didn't support 1.0 last time I checked.

**Pull request merged**